src/main/java/com/uwetrottmann/tmdb2/services/CollectionService.java

```diff
@@ -22,28 +25,62 @@
 import retrofit2.http.GET;
 import retrofit2.http.Path;
 import retrofit2.http.Query;
+import retrofit2.http.QueryMap;
+
+import java.util.Map;
 
 public interface CollectionService {
     /**
      * Get the basic collection information for a specific collection id.
      *
-     * @param tmdbId TMDb id.
-     * @param language <em>Optional.</em> ISO 639-1 code.
+     * @param collectionId TMDb id.
```

Comment:
This comment was about:

![image](https://cloud.githubusercontent.com/assets/888674/26436404/8a5997b2-4116-11e7-9f14-da1147117b95.png)